src/main/java/com/moodybluez/enterprise/dto/Mood.java

```diff
@@ -5,5 +5,5 @@
 public @Data
 class Mood {
     private int moodID;
-    private String moodDesc;
+    private String mood;
```

Comment:
Updated to more generic naming Convention format.
Citation - "a name that tells the reader of the program what the variable represents, Keep the variable names short avoid including metadata" - R. Hamedy

Reply:
Good thought, though I'd call it description.  Having an attribute called mood inside of a class called Mood is confusing.